.github/workflows/master_build_merge.yml

```diff
@@ -182,16 +182,6 @@ jobs:
             npm publish   
             set +e
           }
-          validateInputs() {
```

Comment:
There are more `validateInputs()` definition in this file. Remove all.
Also can we do this scanning for all functions -> If a function is present in `functions-for-build` and if the file is getting sourced for that runner, remove the method definition that is inline - assuming code is same.